build.cake

```diff
@@ -558,7 +558,10 @@ void RunNUnitTests(DirectoryPath workingDir, string testAssembly, string framewo
     try
     {
         var path = workingDir.CombineWithFilePath(new FilePath(testAssembly));
-        NUnit3(path.ToString());
+        var settings = new NUnit3Settings();
+        if(!IsRunningOnWindows())
```

Comment:
I added this in to see what would happen if we ran in-process on Linux. I looked at the failed Travis runs and it looked like there was an issue launching the agent, so I switched to in-process to see what would happen. I also figured that it is no different than running in-process in the nunitlite runner.

It would be interesting to try to debug this on a Linux box. Cake just spawns a process, so their could be a setting in `ProcessSettings` they are missing, or it could be a Mono problem?

I am not wed to this, I could also just spawn the console like we used to.